.github/renovate.json5

```diff
@@ -1,9 +1,17 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:recommended",
+    "docker:pinDigests",
```

Comment:
I think this answers my other [question](https://github.com/open-telemetry/opentelemetry-java/pull/7074#discussion_r1945572671)?